app/scripts/views/sign_up.js

```diff
@@ -48,6 +48,10 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
     },
 
     beforeRender: function () {
+      var self = this;
+      require(['passwordcheck'], function (passwordcheck) {
```

Comment:
Do we need a promise here? if i move the return function inside the require() function, wouldnt it be returned after the library is loaded? Require calls arent thenable I suppose. @shane-tomlinson